src/scss/components/_carousel.scss

```diff
@@ -156,6 +156,12 @@
 		}
 	}
 
+  &--body {
```

Comment:
why do we need of it?

Reply:
I changed to black variant, its needed to set indicator active color